lib/Backend/Func.cpp

```diff
@@ -1552,6 +1555,128 @@ Func::EnsureFuncEndLabel()
     return m_funcEndLabel;
 }
 
+StackArgWithFormalsTracker * 
```

Comment:
Done.

---
In reply to: [68142569](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68142569) [](ancestors = 68142569)